istio/tests/cases/README.md

```diff
@@ -0,0 +1,170 @@
+### Prerequisites
```

Comment:
explain what this is. "This is to run end to end test suite...."

Reply:
ok